src/common/options.cc

```diff
@@ -4687,9 +4687,12 @@ std::vector<Option> get_global_options() {
 
     Option("bluestore_volume_selection_policy", Option::TYPE_STR, Option::LEVEL_DEV)
     .set_default("use_some_extra")
-    .set_enum_allowed({ "rocksdb_original", "use_some_extra" })
+    .set_enum_allowed({ "rocksdb_original", "use_some_extra", "without_slow_dir" })
```

Comment:
My take here is that if we are trying to maximally scale levels to fit on the fast device, we should maybe use something like "fit_to_fast".  Not sure if that's clearer or not.

Reply:
I like this idea. "fit_to_fast" and "always_try_db" are both good options.

Reply:
let it be "fit_to_fast" then

Reply:
I changed the name. It may appear that all the commits are new, but I only added my collegue to the signed-off.